forge/db/views/Project.js

```diff
@@ -210,6 +210,39 @@ module.exports = function (app) {
         return result
     }
 
+    app.addSchema({
+        $id: 'DashboardInstanceSummary',
+        type: 'object',
+        properties: {
+            id: { type: 'string' },
+            name: { type: 'string' },
+            url: { type: 'string' },
+            createdAt: { type: 'string' },
+            updatedAt: { type: 'string' },
+            links: { $ref: 'LinksMeta' },
+            application: { type: 'object', additionalProperties: true },
```

Comment:
this is exposing the raw id

![image](https://github.com/user-attachments/assets/bab710d8-57df-4807-ad29-2cabe63b4e1f)